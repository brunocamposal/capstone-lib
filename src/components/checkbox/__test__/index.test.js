import React from 'react'
import Checkbox from '../index'
import renderer from 'react-test-renderer'

it('should render a checkbox', () => {
  const tree = renderer
    .create(
      <>
        <Checkbox
          type='checkbox'
          className='caps-lib-checkbox'
          name='label'
          value='value'
          onClick={() => {}}
          checked
        />
        label
      </>
    )
    .toJSON()

  expect(tree).toMatchSnapshot()
})
