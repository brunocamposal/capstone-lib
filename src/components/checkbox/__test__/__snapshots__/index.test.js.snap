// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a checkbox 1`] = `
Array [
  <label>
    <input
      checked={true}
      className="sc-bdnylx jyWKkz"
      onClick={[Function]}
      type="checkbox"
      value="value"
    />
  </label>,
  "label",
]
`;
